libs/ledger-live-common/src/env.ts

```diff
@@ -80,6 +80,21 @@ const envDefinitions = {
     def: "https://filecoin.coin.ledger.com",
     desc: "Filecoin API url",
   },
+  API_NEAR_ARCHIVE_NODE: {
+    def: "https://near.coin.ledger.com/node/",
+    parser: stringParser,
+    desc: "Archive node endpoint for NEAR",
+  },
+  API_NEAR_INDEXER: {
+    def: "https://near.coin.ledger.com/indexer/",
+    parser: stringParser,
+    desc: "Datahub Indexer API for NEAR",
+  },
+  API_NEAR_WALLET_INDEXER: {
+    def: "https://api.kitwallet.app/",
```

Comment:
Yes this is an indexer run by the NEAR team! We are using it to fetch the current staking positions for any given account.

Actually, last I heard, a RP was being set up for it by the Ledger infra team - I can follow up with Roman in #near-figment to check on the status of it, because this will need to be swapped out with it.